rotkehlchen/globaldb/assets_management.py

```diff
@@ -0,0 +1,125 @@
+    - InputError: If the version of the file is not valid for the current
+    globaldb version
+    """
+    globaldb = GlobalDBHandler()
+    with open(path) as f:
+        data = ExportedAssetsSchema().loads(f.read())
+
+    # First assert that the version is correct
+    if int(data['version']) != GLOBAL_DB_VERSION:
+        raise InputError('Provided file is for a different version of rotki')
+
+    identifiers = []
+    for asset_data in data['assets']:
+        # Check if we already have the asset with that name and symbol. It is possible that
+        # we have added a missing asset
+        asset_type = asset_data['asset_type']
+        asset_ref: Union[Optional[List[str]], Optional[AssetData]]
+        if asset_type == AssetType.ETHEREUM_TOKEN:
+            asset_ref = globaldb.get_asset_data(
+                identifier=asset_data['identifier'],
+                form_with_incomplete_data=True,
+            )
+        else:
+            asset_ref = globaldb.check_asset_exists(
+                asset_type=asset_type,
+                name=asset_data['name'],
+                symbol=asset_data['symbol'],
+            )
+        if asset_ref is not None:
+            msg_aggregator.add_warning(
+                f'Tried to import existing asset {asset_data["identifier"]} with '
+                f'name {asset_data["name"]}',
+            )
+            continue
+
+        try:
+            globaldb.add_asset(
+                asset_id=asset_data['identifier'],
+                asset_type=asset_type,
+                data=asset_data['extra_information'],
+            )
+        except InputError as e:
+            log.error(
+                f'Failed to import asset with {asset_data["identifier"]=}',
+                f'{asset_type=} and {asset_data=}. {str(e)}',
+            )
+            msg_aggregator.add_warning(
+                f'Failed to save import with identifier '
+                f'{asset_data["identifier"]}.Check logs for more details',
+            )
+            continue
+        identifiers.append(asset_data['identifier'])
+
+    db_handler.add_asset_identifiers(identifiers)
+
+
+def export_assets_from_file(
+    dirpath: Optional[Path],
+    db_handler: 'DBHandler',
+) -> Path:
+    """
+    Creates a zip file with a json file containing the assets added by the user.
+    May raise:
+    - PermissionError if the temp file can't be created
+    """
+    if dirpath is None:
+        dirpath = Path(tempfile.TemporaryDirectory().name)
+        dirpath.mkdir(parents=True, exist_ok=True)
+
+    assets = GlobalDBHandler().get_user_added_assets(user_db=db_handler)
+    serialized = []
+    for asset_identifier in assets:
+        try:
+            asset = Asset(asset_identifier)
+            serialized.append(asset.to_dict())
+        except UnknownAsset as e:
+            log.error(e)
+
+    cursor = GlobalDBHandler()._conn.cursor()
+    query = cursor.execute('SELECT value from clean_db.settings WHERE name=="version";')
```

Comment:
nitpick: We use the `=` operator in all of our sqlite statements. But it's a nitpick as it's exactly the same.

```suggestion
    query = cursor.execute('SELECT value from clean_db.settings WHERE name="version";')
```